### PR TITLE
LPS-119574 Fix top margins of sidebar tabs

### PR DESCRIPTION
**Pull request by rebsilva**

### CODE REVIEW

File: modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -17,6 +17,12 @@
 		max-width: 808px;
 	}
 
+	.sidebar-body {
+		.container-fluid {
```
[the remark this answers is not in the document]
Comment (rebsilva) [reply modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/css/main.scss:21]: Tá certo Keven, farei isso!

**Comment by rebsilva** [comment]
ci:test:sf

**Comment by rebsilva** [comment]
ci:test:relevant

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**